react/RuiExpandCollapseReact/README.md

```diff
@@ -0,0 +1,54 @@
+# Expand Collapse
+
+## Internally controlled
+If you do not need to implement any custom logic, then you just need to provide the component an element for the summary content and the details content that is revealed on expand/collapse. Clicking the summary content will trigger an expand/collapse.
+
+```html preview
+<RuiExpandCollapse>
```

Comment:
Hey Josh,

Out of curiousity, is this pattern possible with the web component approach?

```html
<RuiExpandCollapse open={true}>
  {(isOpen) => (
    <p slot="summary-content">Expander summary. {isOpen ? 'OPEN' : 'CLOSED'}</p>
    <p slot="details-content">Expanded content</p>
  )}
</RuiExpandCollapse >
```

Reply:
@jeffdowdle - It may be possible, I think we'd have to play around with it to know for sure. To make something like that work, we would have to externally control the rui-expand-collapse within the React wrapper by default using an open state. Once we have added the open state to the RuiExpandCollapse react component then we could expose the variable to its children. 

Where I imagine this getting tricky is how we manage externally controlling the react component because now you have this internal state to deal with. Perhaps exposing isOpen negates the need to externally control the component, but I'm not sure.

Reply:
I guess this is something you could build on top of this component, similar to the `ExpandCollapseController` example you have below.

There's lots of ways you could do this, having an internal state that gets overridden when a prop changes is one of them. As long as that kind of render prop is technically possible with web components I'm happy.